### PR TITLE
Plumb flag to skip post-import steps

### PR DESCRIPTION
**Pull request by travisgrigsby**
Need a way from cron.py to skip everything post-import. 
@iandees need your eyeballs to vet my scriptfu.
@peitili I need your eyeballs to make sure I'm passing the --skip-snapshot flag the way you intended.

### CODE REVIEW

**Comment by travisgrigsby** [comment]
👀 @iandees @peitili #maps-basemap-backend-dev

File: batch-setup/provision.sh [hunk 1]
```diff
@@ -97,6 +97,7 @@ export METATILE_SIZE='%(metatile_size)d'
 
 export NUM_DB_REPLICAS='%(num_db_replicas)d'
 export MAX_VCPUS='%(max_vcpus)d'
+export RUN_POST_IMPORT_STEPS='%(run_post_import_steps)s'
```
Comment (travisgrigsby) [inline batch-setup/provision.sh:100]: I'm assuming this comes in as a Python attempt at converting a bool into a string so it should be "True" or "False"... right?

Comment (peitili) [reply batch-setup/provision.sh:100]: how is run_post_import_steps get its value? maybe you need to edit this dict https://github.com/tilezen/tileops/blob/94ca72863e36940a0b2b9dd5bf96ecdeb44eecfb/batch-setup/cron.py#L607

File: batch-setup/cron.py [hunk 2]
```diff
@@ -529,6 +529,8 @@ def is_power_of_two(x):
                         '<planet-url>.md5 convention to specify an md5 '
                         'file for the OSM planet file specified by the '
                         'planet-url argument.')
+    parser.add_argument('--skip-post-import-steps', dest='run_post_import_steps', action='store_false')
+    parser.set_defaults(run_post_import_steps=True)
```
Comment (iandees) [inline batch-setup/cron.py:533]: `action='store_false'` will set default to `True` for you. [see doc](https://docs.python.org/2/library/argparse.html#action)

```suggestion
    parser.add_argument('--skip-post-import-steps', dest='run_post_import_steps', action='store_false')
```

**Review by iandees — APPROVED** [review]

File: batch-setup/provision.sh (hunk 1 above)
Comment (iandees) [reply batch-setup/provision.sh:100]: Yes, `$(run_post_import_step)s` will stringify your boolean into `True` or `False`.

File: batch-setup/cron.py (hunk 2 above)
Comment (travisgrigsby) [reply batch-setup/cron.py:533]: oh interesting.  So I can just leave that out.  Thx!

Comment (travisgrigsby) [reply batch-setup/cron.py:533]: That is a fascinating bit of user friendliness.

File: batch-setup/provision.sh (hunk 1 above)
Comment (travisgrigsby) [reply batch-setup/provision.sh:100]: @peitili - I did edit the dict!

**Review by peitili — APPROVED** [review]

**Pull request merged**